website/docs/flow/index.md

```diff
@@ -1,80 +1,100 @@
 ---
-title: Overview
+title: Flows
 ---
 
-Flows are a method of describing a sequence of stages. A stage represents a single verification or logic step. They are used to authenticate users, enroll them, and more.
+Flows are a major component in authentik. In conjunction with stages and [policies](../policies/index.md), flows are at the heart of our system of building blocks, used to define and execute the workflows of authentication.
+
+There are over a dozen default, out-of-the box flows available in authentik. Users can decide if they already have everything they need with the default flows or if they want to [create](#create-a-custom-flow) their own custom flow, using the Admin interface.
+
+A flow is a method of describing a sequence of stages. A stage represents a single verification or logic step. By connecting a series of stages within a flow (and optionally attaching policies as needed) you can build a highly flexible process for authenticating users, enrolling them, and more.
 
 For example, a standard login flow would consist of the following stages:
 
--   Identification, user identifies themselves via a username or email address
--   Password, the user's password is checked against the hash in the database
--   Log the user in
+-   **Identification stage**: user identifies themselves via a username or email address
+-   **Password stage**: the user's password is checked against the hash in the database
+
+When these stages are successfully completed, authentik logs in the user.
 
-Upon flow execution, a plan containing all stages is generated. This means that all attached policies are evaluated upon execution. This behaviour can be altered by enabling the **Evaluate when stage is run** option on the binding.
+![](./simple_stages.png)
+
+Upon flow execution, a _flow plan_ containing all stages is generated. This means that all attached policies are evaluated upon execution. This behaviour can be altered by enabling the **Evaluate when stage is run** option on the binding. For more information about flow plans, read our [flow context documentation](../flow/context/index.md).
```

Comment:
Our default since quite a bit of time is that policies are not evaluated on plan but rather when the stage is run

Reply:
Ah yes, I knew that but didn't catch it when working on that [older] section. I'll edit it... please review if I got it right this time, thanks.